frontend/src/Pages/LoginSuccess.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useSearchParams } from 'react-router-dom'
+import axios from 'axios';
+
+export default function LoginSuccess() {
+    const [searchParams, setSearchParams] = useSearchParams();
+    const loginToken = searchParams.get('loginToken');
+    const homeServer = localStorage.getItem('homeServer');
+    const fullUrl = new URL('/_matrix/client/v3/login', homeServer);
+
+    async function fetchData(){
+        const response = await axios.post(fullUrl, {
+            type: "m.login.token",
+            token: loginToken
+        });
+        console.log(response.data);
```

Comment:
Assuming this is debug code for now? :)

Even if it is, having a message associated with the log will help determine where it is coming from
```suggestion
        console.log("Matrix server login response:", response.data);
```

Reply:
Removed it :)